src/main/webapp/content/scss/global.scss

```
@@ -1483,16 +1483,45 @@ Note: This class assumes that the scrollbar is already present (e.g. overflow-y
+    background: transparent !important;
+    border: none !important;
+    box-shadow: none !important;
+    border-radius: 0 !important;
+    max-width: 100% !important;
+    max-height: 100% !important;
+    width: 100vw !important;
+    height: 100vh !important;
+    margin: 0 !important;
+    top: 0 !important;
+    left: 0 !important;
+    // Neutralize any PrimeNG enter/leave transform on the dialog root: a transformed ancestor would
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)